UKHomeOffice/sgmr-service
# Cg 121 manifest

I CAN SEE A LIST OF PEOPLE I’VE SAVED TO MY VOYAGE
- Create manifest view
- Update people page and confirm page to show manifest

TEST
Note: there is hard coded data to show how the layout works
- view/create a voyage report
- go to page 4
> you should see the manifest
- go to the check page
> you should see the manifest

——
